pds_doi_service/core/actions/draft.py

```diff
@@ -225,23 +226,24 @@ def _draft_input_files(self, inputs):
             # For each name found, transform the PDS4 label to an OSTI record,
             # then concatenate that record to o_doi_label to return.
             for input_file in list_of_inputs:
-                doi_label = self._run_single_file(
+                doi_labels = self._run_single_file(
                     input_file, self._node, self._submitter, self._force,
                     self._keywords
                 )
 
-                # It is possible that the value of doi_label is None if the file
+                # It is possible that doi_labels is empty if the file
                 # is not a valid label.
-                if not doi_label:
+                if not doi_labels:
                     continue
 
                 # Concatenate each label to o_doi_labels to return.
-                doc = etree.fromstring(doi_label.encode())
+                for doi_label in doi_labels:
+                    doc = etree.fromstring(doi_label.encode())
```

Comment:
Any chance there'll be relative external entities in the `doi_label`? If so, we'll want to pass a `base_url` to `fromstring` to indicate what the original URL is of the XML document. (This only applies to DTDs, XML PIs, XIncludes—so probably a low-to-zero risk of that.)

Reply:
From what I've seen, the OSTI XML doesn't specify anything like that. That, and we'll only be stuck with OSTI's XML format for another couple months so I'm not too concerned